lib/jira/core.rb

```diff
@@ -61,17 +60,10 @@ def ticket?(ticket, verbose=true)
       ### Relevant Paths
 
       #
-      # @return [String] path to .jira-url file
+      # @return [String] path to .jira-cli file
       #
-      def url_path
-        @url_path ||= self.root_path + "/.jira-url"
-      end
-
-      #
-      # @return [String] path to .jira-auth file
-      #
-      def auth_path
-        @auth_path ||= self.root_path + "/.jira-auth"
+      def cli_path
+        @cli_path ||= self.root_path + "/.jira-cli"
```

Comment:
I believe there should be a file for this CLI. Why? More options maybe customizable rather than just site, username, and password. For instance, the format settings could be changed. Another example is for OAuth instead of basic auth. Perhaps ENV variables are good for basic auth so that the username and password isn't stored in a file.